src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -237,15 +222,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuild
       Projects="@(ValidProjectInputForPackGraph)"
       Targets="_AddMetadataToProjectReference"
-      Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration);
-              %(_MSBuildProjectReferenceExistent.SetPlatform);
-              CustomAfterMicrosoftCommonTargets=$(MSBuildThisFileFullPath);
+      Properties="CustomAfterMicrosoftCommonTargets=$(MSBuildThisFileFullPath);
               BuildProjectReferences=false;
-              TargetFramework=$(TargetFramework);
+              _ParentTargetFramework=$(TargetFramework);
```

Comment:
Also, if you're depending on CustomAfterMicrosoftCommonTargets for this, you likely also need CustomAfterMicrosoftCommonCrossTargetingTargets (check spelling).